protecode/util.py

```diff
@@ -25,19 +28,15 @@
 
 def sync_results_with_delivery_db(
     delivery_client: delivery.client.DeliveryServiceClient,
-    results: typing.Iterable[pm.BDBAScanResult],
-    bdba_cfg_name: str,
+    results: collections.abc.Iterable[dso.model.ArtefactMetadata],
```

Comment:
do we actually still need this function?

Reply:
No you're right, actually we can drop this wrapper by now :-)